modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-live-migration-about_{context}"]
+= Live migration to the OVN-Kubernetes network plugin overview
+
+The live migration method is the process in which the OpenShift SDN network plugin and its network configurations, connections, and associated resources, are migrated to the OVN-Kubernetes network plugin without service interruption. It is available for {product-title}, {product-dedicated}, {product-rosa}, and Azure Red Hat OpenShift deployment types It is not available for HyperShift deployment types. This migration method is valuable for deployment types that require constant service availability and offers the following benefits:
+
+* Continuous service availability
+* Minimized downtime
+* Automatic node rebooting
+* Seamless transition from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin.
+
+Although a rollback procedure is provided, the live migration is intended to be a one-way process.
+
+////
+[IMPORTANT]
+====
+Migrating to or from the OVN-Kubernetes network plugin is not supported for managed OpenShift cloud services such as {product-dedicated}, Azure Red Hat OpenShift(ARO), and Red Hat OpenShift Service on AWS (ROSA).
+====
+////
+
+include::snippets/sdn-deprecation-statement.adoc[]
+
+The following sections provide more information about the live migration method.
+
+[id="supported-platforms-live-migrating-ovn-kubernetes"]
+== Supported platforms when using the live migration method
+
+The following table provides information about the supported platforms for the live migration type.
```

Comment:
We shall mention that both IPI and UPI deployments are supported.

Reply:
@pliurh Is it 

```suggestion
|===
| Platform              | Live Migration

| Bare metal hardware (IPI and UPI)             |&#10003;
```

OR 
```
|===
| Platform              | Live Migration
...
| IPI             |&#10003;
| UPI            |&#10003;
```

Reply:
We support both IPI and UPI for all platforms. So the first one looks better, but not only for baremetal.